pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -410,15 +559,17 @@ def expected_purchases_new_customer(
                "A Note on Deriving the Pareto/NBD Model and Related Expressions."
                http://brucehardie.com/notes/009/pareto_nbd_derivations_2005-11-05.pdf
         """
-        # mypy requires explicit typing declarations for these variables.
-        r: xarray.DataArray
-        alpha: xarray.DataArray
-        s: xarray.DataArray
-        beta: xarray.DataArray
+
+        # TODO: If requested, this is still possible from a what-if standpoint.
+        if self.pr_covar_columns or self.pr_covar_columns is not None:
+            raise NotImplementedError(
+                "New customer estimates not currently supported with covariates"
+            )
 
         t = np.asarray(t)
 
-        r, alpha, s, beta = self._unload_params()
+        # TODO: _unload_params requires modification for covariates w/ new customers
```

Comment:
is this TODO part of the PR?

Reply:
It is not. Supporting new customer estimates with covariates will be involved (more details in my original message.) I'll hold off on adding this until there is a demand for it.